### PR TITLE
Happy new year :)

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: ouroboros-consensus-byron-test/NOTICE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2019-2022 Input Output (Hong Kong) Ltd.
+Copyright 2019-2022 Input Output Global Inc (IOG)
```
Comment (bolt12) [inline ouroboros-consensus-byron-test/NOTICE:1]: Should this be 2023 as well?

File: ouroboros-consensus/NOTICE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright 2019-2022 Input Output (Hong Kong) Ltd.
+Copyright 2019-2022 Input Output Global Inc (IOG)
```
Comment (bolt12) [inline ouroboros-consensus/NOTICE:1]: What about here?

**Review by bolt12 — APPROVED** [review]

File: ouroboros-consensus-byron-test/NOTICE (hunk 1 above)
Comment (bolt12) [reply ouroboros-consensus-byron-test/NOTICE:1]: Was this changed in a different commit?

**Review by bolt12 — CHANGES_REQUESTED** [review]

File: ouroboros-consensus-byron-test/NOTICE (hunk 1 above)
Comment (coot) [reply ouroboros-consensus-byron-test/NOTICE:1]: it seems not, done now.

File: ouroboros-consensus/NOTICE (hunk 2 above)
Comment (coot) [reply ouroboros-consensus/NOTICE]: Fixed.

**Review by bolt12 — APPROVED** [review]

**Review by amesgen — APPROVED** [review]
Happy new year :tada:

**Pull request merged**